src/answers-headless.ts

```diff
@@ -165,21 +165,11 @@ export default class AnswersHeadless {
     return response;
   }
 
-  async executeUniversalAutoComplete(): Promise<AutocompleteResponse | undefined> {
-    const thisRequestId = this.httpManager.updateRequestId('universalAutoComplete');
+  async executeUniversalAutoComplete(): Promise<AutocompleteResponse> {
```

Comment:
While we're here, can we rename this to `executeUniversalAutocomplete`? Autocomplete is one word, so the 'C' should not be capitalized. In Core we don't capitalize the 'C'